apps/src/templates/teacherHomepage/GradientNavCard.story.jsx

```diff
@@ -18,7 +10,11 @@ export default storybook => {
         description: `This is an example navigation card with stub data.`,
         story: () => (
           <GradientNavCard
-            cardData={EXAMPLE_CARD_DATA}
+            title= "Teacher Community"
+            description="Ask questions about curriculum, share ideas from your lessons, and get help from other teachers"
+            image="../../static/navcard-placeholder.png"
+            buttonText= "Connect Today"
+            link= "link to teacher community"
```

Comment:
I would still like to see a sample URL at some point so that we know the item works as a link :)